### DISCUSSION TITLE
Constant icon color.

### DISCUSSION THREAD

**Discussion by fcassata** [nicufarmache/lovelace-big-slider-card#49] [Q&A] [answered]
Thanks for making this card available, absolutely love it! Is there a way for me to keep the icon color constant, and not have the opacity change with the brightness of the connected light entity?

**Comment by nicufarmache** [comment, marked as answer]
Right now there is no way to do that. I'm going to try to include an option in the next version

**Reply by fcassata** [reply]
Thanks for the reply :)
